python/llm/test/benchmark/csv_to_html.py

```diff
@@ -39,18 +39,54 @@ def main():
     if len(csv_files)>1:
         data2 = pd.read_csv(csv_files[1], index_col=0)
 
+        data1_df=pd.DataFrame(data1)
+        data2_df=pd.DataFrame(data2)
+
+        last1=[]
+        diff1=[]
+        last2=[]
+        diff2=[]
+
         origin_column_1='1st token avg latency (ms)'
         origin_column_2='2+ avg latency (ms/token)'
 
+        for data1_ind,data1_row in data1_df.iterrows():
+            data1_model=data1_row['model'].strip()
+            data1_input_output_pairs=data1_row['input/output tokens'].strip()
+            flag=True
+
+            for data2_ind,data2_row in data2_df.iterrows():
+                data2_model=data2_row['model'].strip()
+                data2_input_output_pairs=data2_row['input/output tokens'].strip()
+                if data1_model==data2_model and data1_input_output_pairs==data2_input_output_pairs:
+                    data1_temp=data1_row[origin_column_1]
+                    data2_temp=data2_row[origin_column_1]
+                    data3_temp=data1_row[origin_column_2]
+                    data4_temp=data2_row[origin_column_2]
```

Comment:
make the variable names more meaningful.

Reply:
Sure